test/normalize.js

```diff
@@ -143,7 +144,7 @@ tap.test("gist bugs url", function(t) {
     repository: "git@gist.github.com:123456.git"
   }
   normalize(d)
-  t.same(d.repository, { type: 'git', url: 'git@gist.github.com:123456.git' })
+  t.same(d.repository, { type: 'git', url: 'git+ssh://git@gist.github.com/123456.git' })
```

Comment:
Given that npm can't actually understand the ssh connection string form, and it CAN understand the git+ssh URL form, this change in normalization seems desirable.